loadtest/control/simulcontroller/actions.go

```diff
@@ -950,11 +950,9 @@ func unreadCheck(u user.User) control.UserActionResponse {
 }
 
 func searchChannels(u user.User) control.UserActionResponse {
-	team, err := u.Store().CurrentTeam()
+	team, err := u.Store().RandomTeam(store.SelectMemberOf)
```

Comment:
Isn't this a slightly breaking change in logic for non supported versions? May be worth keeping both calls under the version check.